src/index.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+export {default as ActionList} from './components/action-list/ActionList';
```

Comment:
What about all other exports?

Reply:
We don't need to import constants anymore. In all new code, we just use literals instead. Since this `index.js` will be probably used in new code, it's better not to re-export them

Reply:
Are we plannig to run a codemod to remove all the old usages?

Reply:
i'd like to do it eventually, but not sure how time-consuming will it be